plugin.xml

```diff
@@ -63,6 +63,7 @@
     <source-file src="src/ios/GooglePlus.m"/>
 
     <preference name="REVERSED_CLIENT_ID" />
+    <preference name="WEB_APPLICATION_CLIENT_ID" />
```

Comment:
Shouldn't this be in the `<browser>..</browser>` section of plugin.xml? At the moment it's in the `<ios>..</ios>` section.

Reply:
you are correct. that should inside `browser` for more readability. thanks. I updated it